README.md

```diff
@@ -276,12 +276,12 @@ Now that you have created two tables, you'll see entries for them in the `pointc
 > 
 >     1     
  
-**PC_Envelope(p pcpatch)** returns **bytea**
+**PC_Envelope_AsBinary(p pcpatch)** returns **bytea**
```

Comment:
Sorry, I realize this is not very clear. I would just not expose and advertise `PC_Envelope_AsBinary` in the doc but I understand that we need this function for the implementation of `PC_Envelope`. Sorry for not being clear.